core/tee/tee_svc.c

```diff
@@ -313,17 +312,6 @@ static TEE_Result tee_svc_copy_param(struct tee_ta_session *sess,
 					  param_types);
 	}
 
-	if ((called_sess != NULL) &&
```

Comment:
Is this change on purpose in this Pull-Request? It does not look related to ELF / signature changes